atlas/AtlasMl/atlasml/ml/similarity_measures.py

```
@@ -0,0 +1,111 @@
+import numpy as np
+from scipy.spatial.distance import jaccard, euclidean, cosine
+from typing import Union, List
+
+
+def compute_jaccard_similarity(
+    embedding_vector: Union[List[float], np.ndarray], 
```

Ruff (0.12.2)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)